src/components/footer/footer.js

```diff
@@ -1,34 +1,46 @@
 import React from 'react';
-import GlobalStyle from 'global.css.js';
 import { Container } from './footer.css';
+import EmailListForm from 'components/EmailListForm';
+
+//Import social icons and share components
+import { FaTwitter, FaEnvelope, FaFacebook } from 'react-icons/fa';
+import { ShareButtonCircle, ShareBlockStandard } from 'react-custom-share';
+
+const shareBlockProps = {
+  button: ShareButtonCircle,
```

Comment:
the colors stand out a fair amount because the whole page is so green-themed. If easy to make them all more an EDGI green, let's? But if it's a pain, not worth it.

Reply:
Definitely a pain since I would have to edit this custom library I'm using